src/App.scss

```diff
@@ -1,2 +1,16 @@
 // Importing Global Variables
 @import "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
+@import '~@patternfly/patternfly/sass-utilities/_all';
+@import 'node_modules/@patternfly/react-core/dist/styles/base.css';
+
+:root {
+  // Color Variables
+  --root-background-color: var(--pf-global--BackgroundColor--200);
```

Comment:
I think these variable should be already set by PF? Or am I wrong?

Reply:
@Hyperkid123 Like I commented above, I think the whole section is unnecessary.